Clarify that undoing a shutdown might not be possible (#8010)
matrix-org/synapse: docs/admin_api/shutdown_room.md

the structure can and does change without notice.

First, it's important to understand that a room shutdown is very destructive. Undoing a shutdown is not as simple as pretending it
never happened - work has to be done to move forward instead of resetting the past. In fact, in some cases it might not be possible
to recover at all:

> * If the room was invite only, your users will need to be re-invited.

Reviewer: ```suggestion
* If the room was invite-only, your users will need to be re-invited.
```

> * If the room became empty, it'll be impossible to rejoin.

Reviewer: ```suggestion
* If the room no longer has any members at all, it'll be impossible to rejoin.
```

> * If the room was purged from your server, it may be inaccessible.

Reviewer: I don't think this makes any difference?

Author: fair

> 

Reviewer: also:
```suggestion
 * the first user to rejoin will have to do so via an alias on a different server

```

>   * The first user to rejoin will have to do so via an alias on a different server.

Reviewer: is the change in list level deliberate?

Author: nope, I ran an autoformatter after applying suggestions and it decided that 1 space meant 2. Will push it back
